scss/components/_pause-screen.scss

```diff
@@ -1,11 +1,18 @@
 
 .oo-pause-screen {
+  opacity: 1;
```

Comment:
For consistency, would it be better to use the same autohide animation that we use on the playing screen instead of fading the pause screen?

Reply:
Will do.